pkg/version/version.go

```diff
@@ -22,8 +22,16 @@ import (
 	"runtime/debug"
 )
 
+// version to be set using ldflags:
+// -ldflags "-X sigs.k8s.io/controller-tools/pkg/version.version=1.0.0"
+// falls back to module information is unset
+var version = ""
```

Comment:
Let's set this in our release

The release is done via:
* https://github.com/kubernetes-sigs/controller-tools/blob/4fd18cdbabbff1e0b5f054fa346bd8845a6b7029/.github/workflows/release.yaml
* https://github.com/kubernetes-sigs/controller-tools/blob/d1bc045262c5bf8cab5c50523242cb97d744e78b/Makefile#L160

We should be able to get the tag in the GitHub action like this: https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/blob/1161fd4a1f92315c08bd66e1f697b99b365c140c/.github/workflows/release.yaml#L17-L18

(GITHUB_REF:10 cuts the first 10 characters of GITHUB_REF which leaves exactly the tag)

Reply:
Hi @sbueringer we are working on the corporate CLA as we speak.

In the meantime, if I understood correctly, we use the `RELEASE_TAG` env var at the `Makefile`'s rule `release-binary` I assume the env var will go through and we do not need to repeat it on every line of the rule `release-binary`.

Reply:
> In the meantime, if I understood correctly, we use the RELEASE_TAG env var at the Makefile's rule release-binary I assume the env var will go through and we do not need to repeat it on every line of the rule release-binary.

Yes, that's fine. We just also have to modify the GitHub action (.github/workflows/release.yaml). The example action I linked was from another repository